sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/geckoview_version_v1/metadata.yaml

```diff
@@ -6,3 +6,9 @@ description: |
   reference date is 2020-10-01 will include hours up to midnight of that day.
 owners:
   - amiyaguchi@mozilla.com
+labels:
+  incremental: false
+  schedule: daily
+scheduling:
+  dag_name: bqetl_org_mozilla_fenix_derived
+  parameters: ["submission_date:DATE:{{ds}}"]
```

Comment:
This parameter shouldn't be needed if `date_partition_parameter="submission_date"` since the operator wrapper function adds this already.  I don't know if it breaks if a parameter is defined multiple times.

Reply:
I see: https://github.com/mozilla/telemetry-airflow/blob/76e3b6bd2efaaa0c1b143bf531838174a2652c43/dags/utils/gcp.py#L493-L495